lib/dl_file_uploader_worker_lib/dl_file_uploader_worker_lib/utils/parsing_utils.py

```diff
@@ -160,10 +161,17 @@ def _make_id(self, field: converter_parsing_utils.TResultColumn) -> str:
         return idx_to_alphabet_notation(field["index"])  # type: ignore  # TODO: FIX
 
 
+@attr.s
+class YaDocsFieldIdGenerator(FileUploaderFieldIdGenerator):
```

Comment:
We should discuss this a bit more later
We had to implement alphabet notation for gsheets_v2 to remain consistency with the old gsheets connector, I'm not ready to neither confirm or deny that we need it in YaDocs